src/test/ensemble_config_change_stress.cpp

```diff
@@ -1293,17 +1477,32 @@ class StressPipelineConfigChanges : public TestWithTempDir {
             if (!inputNameCorrect) {
                 return false;
             }
-            bool inputTypeCorrect{inputs.at(pipelineInputName.c_str()).dtype() == tensorflow::DT_FLOAT};
-            EXPECT_TRUE(inputTypeCorrect);
-            if (!inputTypeCorrect) {
-                return false;
-            }
-            bool inputShapeCorrect{isShapeTheSame(
-                inputs.at(pipelineInputName.c_str()).tensor_shape(),
-                {1, 10})};
-            EXPECT_TRUE(inputShapeCorrect);
-            if (!inputShapeCorrect) {
-                return false;
+            if (!isMediapipe) {
+                bool inputTypeCorrect{inputs.at(pipelineInputName.c_str()).dtype() == tensorflow::DT_FLOAT};
+                EXPECT_TRUE(inputTypeCorrect);
+                if (!inputTypeCorrect) {
+                    return false;
+                }
+                bool inputShapeCorrect{isShapeTheSame(
+                    inputs.at(pipelineInputName.c_str()).tensor_shape(),
+                    {1, 10})};
+                EXPECT_TRUE(inputShapeCorrect);
+                if (!inputShapeCorrect) {
+                    return false;
+                }
+            } else {
+                bool inputTypeCorrect{inputs.at(pipelineInputName.c_str()).dtype() == tensorflow::DT_FLOAT};
+                EXPECT_TRUE(inputTypeCorrect);
+                if (!inputTypeCorrect) {
+                    return false;
+                }
+                bool inputShapeCorrect{isShapeTheSame(
+                    inputs.at(pipelineInputName.c_str()).tensor_shape(),
+                    {})};
+                EXPECT_TRUE(inputShapeCorrect);
```

Comment:
As far I see the both if statement look exactly the same besides expected shape. Can we unify and just use expected shape as variable depending on MP/DAG?